core/genesis.go

```diff
@@ -248,9 +250,20 @@ func SetupGenesisBlockWithOverride(db ethdb.Database, genesis *Genesis, override
 		return newcfg, stored, compatErr
 	}
 	rawdb.WriteChainConfig(db, stored, newcfg)
+	if storedcfg.PrivacyEnhancementsBlock == nil && newcfg.PrivacyEnhancementsBlock != nil {
```

Comment:
Can we remove `newcfg.PrivacyEnhancementsBlock != nil` condition?  
It seems already be included in the function `checkAndPrintPrivacyEnhancementsWarning()`

Reply:
removed